pytorch_lightning/trainer/trainer.py

```diff
@@ -102,6 +101,7 @@ def __init__(
             train_percent_check: float = 1.0,
             val_percent_check: float = 1.0,
             test_percent_check: float = 1.0,
+            overfit_pct: float = 0.0,
```

Comment:
I would be vaseful with moving arguments as they can be used as positional by some users...

Reply:
ok, I will revert.